.eslintignore

```diff
@@ -11,3 +11,4 @@ globals.d.ts
 integrations/linkedin/proxy/video-upload.ts
 integrations/linkedin/nango-integrations/linkedin/proxy/video-upload.ts
 integrations/*-sandbox
+scripts/generate-script-readmes.ts
```

Comment:
Got too annoyed with eslint? It's not my repo but it's probably better to not flat out ignore everything :D